src/components/hosts/utils.ts

```diff
@@ -120,6 +120,15 @@ export const hasKnownHost = (cluster: Cluster) =>
 export const getHostname = (host: Host, inventory: Inventory) =>
   host.requestedHostname || inventory.hostname;
 
+export const findHostname = (hostId: string, hosts: Host[] = []): string | null => {
+  const host = hosts.find(({ id }) => id === hostId);
+  if (!host) {
+    return null;
```

Comment:
since `host.id` is still valid corner-case result of this function (see bellow), why not use it even in that case?

Reply:
As agreed I refactored this function so it always returns a value. 
See here 25a9c1865a9448d45fdaed6178ad57c5876c5bc9